app/assets/javascripts/users.js.erb

```diff
@@ -104,18 +106,20 @@ function showTarget(position, radius, icon, targetId, route){
 };
 
 function createTarget(pos) {
-  $('.home-left-new-target').removeClass('hidden');
-  $('#new-target-title').val('');
-  $('#selected_area').val(50);
-  newTarget.setRadius(50);
-  $('.select-topic').html('What do you want to talk about?');
-  $('.topic').removeClass('selected');
-  $('.targets-list').addClass('hidden');
-  $('.home-left').addClass('hidden');
-  $('.home-left-delete-target').addClass('hidden');
-  map.setCenter(pos);
-  newTarget.setCenter(pos);
-  deselectTarget();
+  if (firstLogin) {
+    $('.home-left-new-target').removeClass('hidden');
+    $('#new-target-title').val('');
+    $('#selected_area').val(50);
+    newTarget.setRadius(50);
+    $('.select-topic').html('What do you want to talk about?');
+    $('.topic').removeClass('selected');
+    $('.targets-list').addClass('hidden');
```

Comment:
You can do `$('.targets-list, .home-left, .home-left-delete-target').addClass('hidden')`